src/rpcblockchain.cpp

```diff
@@ -472,15 +490,15 @@ UniValue getblock(const UniValue& params, bool fHelp)
     if(!ReadBlockFromDisk(block, pblockindex))
         throw JSONRPCError(RPC_INTERNAL_ERROR, "Can't read block from disk");
 
-    if (!fVerbose)
+    if (verbosity == 0)
     {
         CDataStream ssBlock(SER_NETWORK, PROTOCOL_VERSION);
         ssBlock << block;
         std::string strHex = HexStr(ssBlock.begin(), ssBlock.end());
         return strHex;
     }
 
-    return blockToJSON(block, pblockindex);
+    return blockToJSON(block, pblockindex, verbosity >= 2);
```

Comment:
Change to `verbosity == 2` to match the earlier change.

Reply:
I see @daira [reasoned it was fine](https://github.com/zcash/zcash/pull/3095#discussion_r176944896); fair enough.